stack/Pulumi.thousand-genomes.yaml

```diff
@@ -0,0 +1,10 @@
+encryptionsalt: v1:SPQXlVggjxw=:v1:X1sTpViNuyK+Wcom:3GANI8gZOKtk0gG7BklsyHeNU5uVLw==
+config:
+  datasets:customer_id: C010ys3gt
+  datasets:enable_release: "false"
+  datasets:hail_service_account_test: thousand-genomes-test-508@hail-295901.iam.gserviceaccount.com
+  datasets:hail_service_account_standard: thousand-genomes-standard-832@hail-295901.iam.gserviceaccount.com
+  datasets:hail_service_account_full: thousand-genomes-full-579@hail-295901.iam.gserviceaccount.com
+  gcp:billing_project: autonomous-tube-321822
+  gcp:project: autonomous-tube-321822
```

Comment:
Do we want this name to more accurately reflect the name of the stack.

Reply:
Unfortunately you can't control the GCP project ID, but just the GCP project name. We could try another project name though.

Reply:
This is fixed now: I was able to create a new GCP project with the ID `thousand-genomes`.